addons/binding/org.openhab.binding.hdanywhere/README.md

```diff
@@ -4,16 +4,18 @@ This binding integrates with [HDanywhere](http://www.hdanywhere.co.uk) HDMI matr
 
 HDanywhere is a manufacturer of multiroom/distributed audio/video equipment.
 This binding supports their V3 of the Multiroom+ HDMI matrix running firmware V1.2(20131222), as well as the newer MHUB series of matrices.
-These matrices support the highest HD resolutions, including 1080p 3D & 4K, use a single Cat5e/6/7 wiring structure with reliable performance up to 100m, have IR passback to allow you to select and control what you watch from every room and are fully compatible with universal remote controls
+These matrices support the highest HD resolutions, including 1080p 3D & 4K, use a single Cat5e/6/7 wiring structure with reliable performance up to 100m,
+have IR passback to allow you to select and control what you watch from every room and are fully compatible with universal remote controls
```

Comment:
This whole section sounds like a commercial for the product.  Delete it?  Minimize the marketing language?